src/Quartz.AspNetCore/Quartz.AspNetCore.csproj

```diff
@@ -1,25 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <Description>Quartz.NET ASP.NET Core integration; $(Description)</Description>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <RootNamespace>Quartz</RootNamespace>
-        <GenerateDocumentationFile>True</GenerateDocumentationFile>
-        <Authors>Lewis Zou, Marko Lahma, Quartz.NET</Authors>
-    </PropertyGroup>
-    <ItemGroup>
-        <Compile Include="..\AssemblyInfo.cs" />
-    </ItemGroup>
+  <PropertyGroup>
+    <Description>Quartz.NET ASP.NET Core integration; $(Description)</Description>
+    <TargetFrameworks>netcoreapp2.2;netstandard2.0</TargetFrameworks>
+    <RootNamespace>Quartz</RootNamespace>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <Authors>Lewis Zou, Marko Lahma, Quartz.NET</Authors>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.6" />
-        <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0" />
-    </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\AssemblyInfo.cs" />
+  </ItemGroup>
 
-    <ItemGroup>
-      <ProjectReference Include="..\Quartz.Extensions.DependencyInjection\Quartz.Extensions.DependencyInjection.csproj" />
-      <ProjectReference Include="..\Quartz.Extensions.Hosting\Quartz.Extensions.Hosting.csproj" />
-      <ProjectReference Include="..\Quartz\Quartz.csproj" />
-    </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
```

Comment:
@lahma multi-targeting is a good option to add support features that are only present in newer versions of the .NET Platform Extensions while still supporting older ones. Not sure if it's something you're interested in doing so let me know if you don't like this approach and I'll revert these changes.

Reply:
I think this looks fine 👍